<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyMouse 数据查看工具 v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_hr_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>心率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hr_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_spo2_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>血氧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_spo2_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mc_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>微循环</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mc_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fi_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>疲劳指数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fi_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sbp_title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>收缩压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sbp_value">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dbp_title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>舒张压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dbp_value">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_co_title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>心输出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_co_value">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pr_title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>外周阻力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pr_value">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>开始体检</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_history">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>历史数据</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>200</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>246</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>移动距离</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_distance">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>246</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>296</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>左键点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_leftclick">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>296</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>296</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>右键点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rightclick">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>296</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>296</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中键点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_midclick">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>296</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mousedata">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
